scipy/stats/stats.py

```diff
@@ -1377,10 +1377,9 @@ def normaltest(a, axis=0, nan_policy='propagate'):
     References
     ----------
     .. [1] D'Agostino, R. B. (1971), "An omnibus test of normality for
-           moderate and large sample size," Biometrika, 58, 341-348
+           moderate and large sample size", Biometrika, 58, 341-348
 
-    .. [2] D'Agostino, R. and Pearson, E. S. (1973), "Testing for
-           departures from normality," Biometrika, 60, 613-622
+    .. [2] D'Agostino, R. and Pearson, E. S. (1973), "Tests for departure from normality. Empirical results for the distributions of b2 and ...", Biometrika, 60, 613-622
```

Comment:
The original title is "Tests for departure from normality. Empirical results for the distributions of b2 and √b1", where the symbol in front of the b1 is the square root symbol. As this is a non-ascii symbol we cannot use it. I do as you suggest and just go with the first sentence of the title